### PR TITLE
Update Context Readme

### PR DESCRIPTION
**Pull request by virkt25**
Connect to #260

cc @bajtos @raymondfeng @ritch @superkhau

### CODE REVIEW

**Comment by bajtos** [comment]
@virkt25 could you please run `git fetch origin && git rebase origin/master && push -f` to update this feature branch to the latest master?

File: packages/context/README.md [hunk 1]
```diff
@@ -1 +1,59 @@
 # @loopback/context
+
+Loopback uses Context to manage state and dependencies in your application.
```
Comment (superkhau) [inline packages/context/README.md]: LoopBack

File: packages/context/README.md [hunk 2]
```diff
@@ -1 +1,59 @@
...
+
+## Overview
+
+- An abstraction of all state and dependencies in your application.
+- Context is what LB uses to "manage" everything.
```
Comment (superkhau) [inline packages/context/README.md]: Remove: This sentence doesn't add anything

File: packages/context/README.md [hunk 3]
```diff
@@ -1 +1,59 @@
...
+- A global registry for anything/everything in your app (all configs, state, dependencies, classes, etc).
+- An IoC container used to inject dependencies into your code.
```
Comment (superkhau) [inline packages/context/README.md]: This section should be a simple paragraph (not bullet list) describing everything above.

LoopBack implements the concept of a Context to represent a global registry in your application to manage an application's state and dependencies...etc

**Comment by loay** [comment]
@slnode test please

**Pull request merged**